.travis.yml

```diff
@@ -92,7 +92,7 @@ install:
   # Install ldoc for building docs.
   - |
     if [ "$BUILD_APIDOC" == "true" ]; then
-      travis_retry sudo luarocks install ldoc 1.4.4
+      travis_retry sudo luarocks install ldoc 1.4.6
```

Comment:
I've thought of luacov with my previous comment.
Pulled out the update of ldoc into https://github.com/awesomeWM/awesome/pull/1334.